glutton.go

```diff
@@ -186,6 +198,8 @@ func (g *Glutton) registerHandlers() {
 // Shutdown the packet processor
 func (g *Glutton) Shutdown() (err error) {
 	defer g.logger.Sync()
+	g.cancel() // close all connection
+	time.Sleep(4 * time.Second)
```

Comment:
That argument is good enough for me :) Keep it as it is, maybe add a TODO here or maybe an issue in github.